### PR TITLE
Created the user interface

### PR DESCRIPTION
**Pull request by greybuz**
I created the user interface. However, I came across a small issue: I'm unsure how to create the "logout" dropdown menu for the user circle in the AgendaFragment. I played around with the Spinner and used a String Array with one element being "Logout", but the Spinner shows the first element ("Logout") upfront, without me even clicking it. Could you help me with that?

### CODE REVIEW

File: app/src/main/java/com/example/taskyapp/EditTitleFragment.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        return inflater.inflate(R.layout.fragment_edit_title, container, false)
+    }
+
+}
```
Comment (philipplackner) [inline app/src/main/java/com/example/taskyapp/EditTitleFragment.kt:23]: The functionality of this fragment is exactly the same as for the description. Only some texts are different, so you're fine to keep one `EditTextFragment`

**Review by philipplackner — APPROVED** [review]

File: app/src/main/java/com/example/taskyapp/EditTitleFragment.kt (hunk 1 above)
Comment (greybuz) [reply app/src/main/java/com/example/taskyapp/EditTitleFragment.kt:23]: You're suggesting I delete one?

File: app/src/main/java/com/example/taskyapp/EventDetailAttendeeEditFragment.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        return inflater.inflate(R.layout.fragment_event_detail_attendee_edit, container, false)
+    }
+
```
[the remark this answers is not in the document]
Comment (greybuz) [reply app/src/main/java/com/example/taskyapp/EventDetailAttendeeEditFragment.kt:24]: Oh. The Mockup Figma screen shows a slightly different screen named "view as attendee editable". You're suggesting I delete my EventDetailAttendeeEditFragment?

File: app/src/main/java/com/example/taskyapp/EventDetailAttendeeFragment.kt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.example.taskyapp
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class EventDetailAttendeeFragment : Fragment() {
```
[the remark this answers is not in the document]
Comment (greybuz) [reply app/src/main/java/com/example/taskyapp/EventDetailAttendeeFragment.kt:9]: So, delete? (just want to make sure before changing code)

File: app/src/main/java/com/example/taskyapp/EventDetailEditFragment.kt [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package com.example.taskyapp
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class EventDetailEditFragment : Fragment() {
```
[the remark this answers is not in the document]
Comment (greybuz) [reply app/src/main/java/com/example/taskyapp/EventDetailEditFragment.kt:9]: I understand. But your screens have slight different views (the + button, the > button etc.), so should i just show and hide those buttons when necessary?

File: app/src/main/java/com/example/taskyapp/SplashScreenFragment.kt [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        return inflater.inflate(R.layout.fragment_splash_screen, container, false)
+    }
+
+}
```
[the remark this answers is not in the document]
Comment (greybuz) [reply app/src/main/java/com/example/taskyapp/SplashScreenFragment.kt:25]: Thank you

File: app/src/main/java/com/example/taskyapp/TaskDetailFragment.kt [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+        @JvmStatic
+        fun newInstance(param1: String, param2: String) =
+            TaskDetailFragment().apply {
+                arguments = Bundle().apply {
+                    putString(ARG_PARAM1, param1)
+                    putString(ARG_PARAM2, param2)
+                }
+            }
+    }
+}
```
[the remark this answers is not in the document]
Comment (greybuz) [reply app/src/main/java/com/example/taskyapp/TaskDetailFragment.kt:59]: Should i edit the code to make it look like my other fragments?

File: app/src/main/java/com/example/taskyapp/EditTitleFragment.kt (hunk 1 above)
Comment (philipplackner) [reply app/src/main/java/com/example/taskyapp/EditTitleFragment.kt:23]: Correct :)

File: app/src/main/java/com/example/taskyapp/EventDetailAttendeeEditFragment.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (philipplackner) [reply app/src/main/java/com/example/taskyapp/EventDetailAttendeeEditFragment.kt:24]: That relates to how the screen looks like if you're an attendee of an event and open event details (then you can't fully edit the event for example)

File: app/src/main/java/com/example/taskyapp/EventDetailAttendeeFragment.kt (hunk 3 above)
[the remark this answers is not in the document]
Comment (philipplackner) [reply app/src/main/java/com/example/taskyapp/EventDetailAttendeeFragment.kt:9]: Yes

File: app/src/main/java/com/example/taskyapp/EventDetailEditFragment.kt (hunk 4 above)
[the remark this answers is not in the document]
Comment (philipplackner) [reply app/src/main/java/com/example/taskyapp/EventDetailEditFragment.kt:9]: You can export vectors from Figma directly by selecting it, right clicking and copying it as SVG. Then paste into an empty file, save it as SVG and import it in Android Studio as XML. You can do that by right clicking on drawable->New->Vector asset and then choosing your SVG file

File: app/src/main/java/com/example/taskyapp/TaskDetailFragment.kt (hunk 6 above)
[the remark this answers is not in the document]
Comment (philipplackner) [reply app/src/main/java/com/example/taskyapp/TaskDetailFragment.kt:59]: Yes :)

**Pull request merged**